tools/ScrollAnimationAssembler/create_scroll.dm

```diff
@@ -0,0 +1,51 @@
+
+/mob/verb/Create_Scroll()
```

Comment:
I think it would be much better to have it go something like

* Send a prompt for the very first variable (singular)
* Check the variable for validity
* If invalid, throw an error and exit.
* If valid, only then move on to the next variable

Feels like bad UX to have someone type in five different values, and not find out they screwed up the second one until the very end.